bpmn/Dockerfile

```diff
@@ -26,13 +26,19 @@ RUN /usr/local/bin/pkg --targets ${NODE}-${PLATFORM}-${ARCH} . -o app.bin
 # Create the image
 FROM alpine:3.13.2
 
-COPY --from=builder /app/app.bin /node/bin/bpmn
-COPY assets /app/assets
+RUN apk add --no-cache --update chromium
+
+RUN addgroup -g 1000 kroki && adduser -D -G kroki -u 1000 kroki
+
+USER kroki
+WORKDIR /home/kroki
```

Comment:
I wonder if `WORKDIR` is needed here? We are only using absolute path so nothing gets added in `/home/kroki`?